app/react-components/components/FrequentlyMutatedGenesContainer.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { withState, withProps, lifecycle, compose } from 'recompose';
+import FrequentlyMutatedGenes from './FrequentlyMutatedGenes';
+import { PaginationContainer } from '../uikit/Pagination';
+
+export default
+compose(
+  withState('state', 'setState', { data: { hits: [], total: 0 }, loading: true }),
+  withProps({
+    fetchData: async props => {
+      props.setState(s => ({ ...s, loading: true }));
+
+      const url =
+        `${props.config.es_host}/${props.config.es_index_version}-gene-centric/gene-centric/_search`;
```

Comment:
we shouldn't be calling `es_host` any more, would we switch this to hit the api?